packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/virtualMachine/dedicatedCloud-datacenter-virtualMachine.component.js

```diff
@@ -11,6 +11,8 @@ export default {
     setMessage: '<',
     goToDeleteLicense: '<',
     goToSetLicense: '<',
+    trackClick: '<',
+    trackPage: '<',
```

Comment:
Not used in the controller
```suggestion
    trackPage: '<',
```